account_journal_activate_tracking/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+{
+    "name": "Account journal - activate tracking",
+    "summary": """
+        Activate tracking on account journal
```

Comment:
Better this IMO (more precise)
```suggestion
        Activate tracking on some account journal fields
```